src/setup-wp-server-files.ts

```diff
@@ -58,14 +62,27 @@ async function copyBundledWPCLI() {
 	const bundledWPCLIPath = path.join( getResourcesPath(), 'wp-files', 'wp-cli', 'wp-cli.phar' );
 	await fs.copyFile( bundledWPCLIPath, getWpCliPath() );
 }
+
+async function copyBundledSQLiteCommand() {
+	const isSqliteCommandInstalled = await fs.pathExists( getSqliteCommandPath() );
+	if ( isSqliteCommandInstalled ) {
+		return;
+	}
```

Comment:
Good point, I'd like to move this to a separate PR as it is not critical. I'll create an issue for it today.

Reply:
Oh, sorry, when I shared [the PR](https://github.com/Automattic/studio/pull/422) I didn't want to relate it to the logic of updating all sites. The part I wanted to highlight is [this one](https://github.com/Automattic/studio/pull/422/commits/3a4e47a25c7d78b2535a93c69aea3caed4e1d2b5), which updates the local version (i.e. the one located in the server files) with the bundled one if it's newer. Otherwise, if the user has a version of SQLite command installed, it won't ever be updated with the bundled one even if it's newer.

Reply:
Gotcha, Thanks for clarifying. I created an issue: https://github.com/Automattic/dotcom-forge/issues/8602

Feel free to add details if I am missing something in the description.